pkg/controller/disruption/disruption.go

```diff
@@ -43,6 +44,12 @@ import (
 
 const statusUpdateRetries = 2
 
+// DeletionTimeout sets maximum time from the moment a pod is added to DisruptedPods in PDB.Status
+// to the time when the pod is expected to be marked for deletion. If the pod was not marked for
```

Comment:
I guess it would be more accurate to say "to the time when the pod is expected to be seen by PDB controller as having been marked for deletion" ?